### ISSUE TITLE
Download documentation

### ISSUE THREAD

**Issue by tenjaa** [davidje13/lean-qr#8]
Hi,
when I look at the example to download a file, I feel like there is something missing.
https://qr.davidje13.com/docs/

```
<a href="#" download="qr.png" id="download-link">Download</a>

<script type="module">
  import { generate } from 'lean-qr';

  const code = generate('LEAN-QR LIBRARY');

  const dataUrl = qrCode.toDataURL({ scale: 10 });
  document.getElementById('download-link')
    .setAttribute('href', dataUrl);
</script>
```

- Where does `qrCode` come from?
- I don't think there is a function `toDataURL` on the returntype of `generate`

**Comment by tenjaa** [comment]
Nevermind...
I had browser native open and was looking for node.

**Comment by tenjaa** [comment]
Investigating a bit more, there seems to be `toDataURL` but this function is not exposed as typescript type. Is this intended?
Also I guess `qrCode` and `code` should be the same thing?

**Comment by davidje13** [comment]
Yes seems to be a typo in the example with `qrCode` vs `code`, and `toDataURL` is missing from the types (but [is documented](https://qr.davidje13.com/docs/#Bitmap2D_toDataURL)). I'll fix both. Thanks for reporting!

**Comment by davidje13** [comment]
Released version 2.3.3 which fixes both issues.
